prisma/schema.prisma

```diff
@@ -18,7 +18,7 @@ model Employee {
   department   Department @relation(fields: [departmentId], references: [departmentId])
   departmentId Int
   projects ProjectAssignment[]
-  profilePictureUrl EmployeeProfilePicture?
+  image EmployeeProfilePicture?
 }
 
 model EmployeeProfilePicture {
```

Comment:
Why is this a separate model?

Reply:
This was the initial Idea. Was not sure on how to implement with employee. So kept it in this way.

Reply:
I can change this and implement with employee now.

Reply:
Implemented within employee.